spec/minitest_helper.rb

```diff
@@ -0,0 +1,21 @@
+ENV['MT_NO_PLUGINS'] = '1' # Work around stupid autoloading of plugins
+gem 'minitest'
```

Comment:
OK, I have never seen such a thing outside a `Gemfile`

Please remove or educate me what it is doing :)

Reply:
Same for me, shouldn't we try to keep gem loading instructions inside `Gemfile` work with something like

```ruby 
group :test do
  gem 'minitest'
end
```